tests/acceptance/smoke.tsx

```diff
@@ -126,7 +126,7 @@ if (Meteor.isClient) {
 
     describe('which are unauthenticated', function () {
       before(async function () {
-        await Meteor.wrapPromise(Meteor.logout);
+        await Meteor.wrapPromise(Meteor.logout)();
```

Comment:
Huh, what was the impact before this change here?  Were we never actually logging out here?  (I vaguely recall we avoid testing routes with a token in them like account-enrollment and password reset, so only `/login` and `/create-first-user` were even candidates for running here)

Reply:
Yeah we would have never actually logged out